server/src/test/java/org/elasticsearch/search/aggregations/bucket/histogram/InternalAutoDateHistogramTests.java

```diff
@@ -108,7 +108,6 @@ public void testGetAppropriateRoundingUsesCorrectIntervals() {
         assertThat(result, equalTo(2));
     }
 
-    @AwaitsFix(bugUrl = "https://github.com/elastic/elasticsearch/issues/39497")
```

Comment:
This is also a mistake, bled over from a WIP PR.  Must have done a checkout from the wrong branch originally or something.

Fixing